### PR TITLE
Initial roadmap

### PR DESCRIPTION
**Pull request by dragan**
Initial roadmap to document vision and direction.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+[Project-Name-Placeholder]. To the point where they can start from scratch in a
+new environment and without our direct help.
+
+## Components
+
+### TODO: Break this out into an architecture and design document
+
+### Quoins
+
+__Quoins__ are terraform modules that tie different aspects of an infrastructure
```
Comment (nicklathe) [inline ROADMAP.md]: Link to https://www.terraform.io/ here?

Comment (nicklathe) [inline ROADMAP.md]: Nevermind, it's linked below.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+- [quoins][quoins] - Used for sharing to the community
+
+[quoins]: https://github.com/[Project-Name-Placeholder]/quoins
+
+### Eve & Evectl
+
+__Eve__ is an HTTP API built in Go and is the central component of Rohr. Eve
+manages context and authentication to create infrastructure and perform
+deployments on behalf of a team. Eve enforces Concur's policies by validating
```
Comment (weitenghuang) [inline ROADMAP.md]: should we not specify ~~Concur's~~ policies?

Comment (dragan) [inline ROADMAP.md]: Nope, I need to scrub that.

**Pull request merged**